app/services/idv/session.rb

```diff
@@ -183,14 +204,6 @@ def address_confirmed!
       session[:gpo_code_verified] = true
     end
 
-    def invalidate_steps_after_ssn!
-      # Guard against unvalidated attributes from in-person flow in review controller
-      clear_applicant!
-
-      invalidate_verify_info_step!
-      invalidate_phone_step!
-    end
-
     def clear_applicant!
```

Comment:
This is still used in `extract_pii_from_doc` in DocumentCaptureConcern. It is already covered by `clear_future_steps!` in DocumentCaptureController#update and LinkSentController#update, so we could remove it.